packages/babel-plugin-emotion/src/utils/transform-expression-with-styles.js

```diff
@@ -86,7 +79,7 @@ export let transformExpressionWithStyles = ({
       path.node.arguments.length === 1 &&
       t.isStringLiteral(path.node.arguments[0])
     ) {
-      let cssString = path.node.arguments[0].value
+      let cssString = path.node.arguments[0].value.replace(/;$/, '')
```

Comment:
trailing semi is not needed as we put a semi after `opaqueCss.styles` at runtime anyway:
https://github.com/emotion-js/emotion/blob/0b817dade11dee342a820d6c59e86f8a7ef23493/packages/serialize/src/index.js#L171